README.md

```diff
@@ -1 +1,221 @@
-Coming soon.
+# EventHandler
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+
+
+- [Document historiek](#document-historiek)
+- [Introductie](#introductie)
+- [Links](#links)
+- [Publish](#publish)
+  - [Create namespace](#create-namespace)
+  - [Create topic](#create-topic)
+  - [Packages](#packages)
+  - [Omgaan met onbeschikbaarheden](#omgaan-met-onbeschikbaarheden)
+- [Subscribe](#subscribe)
+  - [Create subscription](#create-subscription)
+    - [Topic subscription](#topic-subscription)
+    - [Wildcard subscription](#wildcard-subscription)
+  - [Volgorde van berichten](#volgorde-van-berichten)
+  - [Retries strategies](#retries-strategies)
+    - [First level: Success](#first-level-success)
+    - [First level: Delete](#first-level-delete)
+    - [First level: Error Queue](#first-level-error-queue)
+    - [First level: Stop](#first-level-stop)
+    - [Second level: Success, error, delete](#second-level-success-error-delete)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Document historiek
+
+Versie   | Auteur                 | Datum      | Opmerkingen
+------   | -------                | -----      | ------------
+1        | Dries Droesbeke | 04/04/2018 | Initial draft.
+
+# Introductie
+
+De eventhandler is een ACPAAS component die gebruikt wordt om berichten te publiceren waarop verschillende afnemers kunnen subscriben. 
+
+De eventhandler is een abstractielaag voor pub/sub dat momenteel gebruik maakt van RabbitMQ en MongoDB en volledig werkt op basis van http voor het versturen van de berichten.
```

Comment:
... voor pub/sub **die** ...

En is het noodzakelijk om te weten dat dit gebaseerd is op RabbitMQ en mongoDB ?

Reply:
resolved